src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -513,6 +546,13 @@ INSTANTIATE_TEST_SUITE_P(
         ReduceCPULayerTest::getTestCaseName
 );
 
+INSTANTIATE_TEST_SUITE_P(
```

Comment:
Ideally we should explain why such custom test cases is needed. For example here we test post-ops fusing case with U8/I8 precision on output since Transpose prevents dequantization part to be fused back into Reduce.

Reply:
Good point! Description added. Thanks Dmitry!